auction-app-backend/src/main/java/com/example/auctionapp/controller/TestController.java

```diff
@@ -0,0 +1,13 @@
+package com.example.auctionapp.controller;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```

Comment:
Better approach here would be to use @RequestMapping annotation for entire class and set url for it (for example this is TestController, you can do this, @RequestMapping('api/v1/test') and every API you create inside this controller will start with this endpoint). So you don't need to write entire endpoint every time, just additional parameters. 
Also in Spring Boot you have annotations for HTTP methods, so instead of this 
`@RequestMapping(method = RequestMethod.GET, path = "/test")`
You can write this 
`@GetMapping("/test")`